src/mono/wasm/Wasm.Build.Tests/Blazor/BuildPublishTests.cs

```diff
@@ -65,15 +66,26 @@ public void DefaultTemplate_NoAOT_WithWorkload(string config)
     }
 
     [Theory]
-    [InlineData("Debug")]
-    [InlineData("Release")]
-    public void DefaultTemplate_BuildNative_WithWorkload(string config)
+    [InlineData("Debug", false)]
+    [InlineData("Release", false)]
+    [InlineData("Debug", true)]
+    [InlineData("Release", true)]
+    public void DefaultTemplate_CheckFingerprinting(string config, bool expectFingerprintOnDotnetJs)
     {
         string id = $"blz_buildandbuildnative_{config}_{Path.GetRandomFileName()}";
 
         CreateBlazorWasmTemplateProject(id);
 
-        BlazorBuild(new BlazorBuildOptions(id, config, NativeFilesType.Relinked), "/p:WasmBuildNative=true");
+        void Execute(Func<BlazorBuildOptions, string[], (CommandResult, string)> target) => target(
+            new BlazorBuildOptions(id, config, NativeFilesType.Relinked, ExpectFingerprintOnDotnetJs: expectFingerprintOnDotnetJs),
+            new[] {
+                "/p:WasmBuildNative=true",
```

Comment:
This is needed only for build.

Reply:
Sorry, misread the error message. It check whether after publish the `for-build` is empty

Reply:
Ah, that would be the case for Debug. Ok, ignore my comment then.